src/pages/PrepareCondition/index.tsx

```diff
@@ -474,8 +491,11 @@ export const PrepareCondition = () => {
           )}
           {isConditionAlreadyExist && (
             <StatusInfoInline status={StatusInfoType.warning}>
-              Condition already exist. Please use another question ID or change the number of
-              outcomes.
+              Condition{' '}
+              <Link href={`#/conditions/${conditionIdPreview}`} target="_blank">
+                {conditionIdPreview}
```

Comment:
Maybe we can use the shorter version for a hash with dots ? Looks really long and sometimes breaks the `word`
![Screenshot_20201022_091732](https://user-images.githubusercontent.com/1144028/96870547-7306ef80-1447-11eb-8f48-d22e3444de0f.png)

Reply:
Agreed.